docs/content/Dialog2.mdx

```diff
@@ -87,6 +87,7 @@ By default, the Dialog component implements the design and interactions defined
 | role          | `"dialog" │ "alertdialog"`                     | `"dialog"` | The ARIA role given to the dialog element. More info: [dialog](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal), [alertdialog](https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog)                                                                    |
 | width         | `"small" │ "medium" │ "large" │ "xlarge"`      | `"xlarge"` | The fixed width of the dialog.                                                                                                                                                                                                                                          |
 | height        | `"small" │ "large" │ "auto"`                   | `"auto"`   | The fixed height of the dialog, or, auto to adjust the height based on its contents.                                                                                                                                                                                    |
+| sx            | SystemStyleObject                              | {}         | Style to be applied to the component                                                                                                                                                                                                                                    |
```

Comment:
Non-blocking nitpick: I would have put "SystemStyleObject" to show it as preformatted text.

Reply:
Ohh that's a good call. I was just chatting with @colebemis the other day about how we could make some react components to build prop tables in a semi-automated way.
```suggestion
| sx            | `SystemStyleObject`                            | {}         | Style to be applied to the component                                                                                                                                                                                                                                    |
```